src/i18.rs

```diff
@@ -0,0 +1,20 @@
+
+use fluent_resmgr::resource_manager::ResourceManager;
+use sys_locale::get_locale;
+use unic_langid::LanguageIdentifier;
+
+pub fn translate(change: &str) -> String{
+    let locale = get_locale().unwrap_or_else(|| String::from("en-US"));
+
+    let langid: LanguageIdentifier = locale.parse().expect("wrong language"); 
+    let locales = vec![langid.into()];
+    let resources = vec!["message.ftl".into()];
+    let mgr = ResourceManager::new("./translations/{locale}/{res_id}".into()); 
```

Comment:
This part is to obtain international resources, that is, the translation files required for internationalization. The current processing method is to store the translations directory in the /usr/share/fd directory, and modify the code here to the absolute path, that is, "/usr/share/fd/translations/{locale}/{res_id}"

Reply:
> The current processing method is to store the translations directory in the /usr/share/fd directory, and modify the code here to the absolute path, that is, "/usr/share/fd/translations/{locale}/{res_id}"

It would be better if the packaging process doesn't require modifying the code. 

And what if it is just installed from a zip file at an unknown location. How will fd know where the translation files are?

What happens if it can't find the translation files?